src/javaRosa/plugin.js

```diff
@@ -192,8 +192,8 @@ define([
                 }
             });
 
-            if (form && form.vellum.getCurrentlySelectedMug()) {
-                form.vellum.getCurrentlySelectedMug().fire({
+            if (form) {
```

Comment:
Nice I like this change. I wonder when this would happen without a form object though